app/src/main/java/com/duckduckgo/app/browser/BrowserWebViewClient.kt

```diff
@@ -182,7 +181,7 @@ class BrowserWebViewClient @Inject constructor(
     private fun reportHttpsUpgradeSiteError(url: Uri, error: String?) {
         val params = mapOf(
             APP_VERSION to BuildConfig.VERSION_NAME,
-            URL to url.simpleUrl,
+            URL to "${url.scheme}://${url.host}",
```

Comment:
Previously, we had a null check around the `url.scheme` - are we cool with not having that null check any more?

Reply:
We are. We also know we are dealing with a https site at this point so I have updated it to use the https scheme directly.

Reply:
Even better! 👍